community-operators/rook/rook.v0.1.0.csv.yaml

```diff
@@ -0,0 +1,316 @@
+                serviceAccountName: rook-ceph-system
+                volumes:
+                  - emptyDir: {}
+                    name: rook-config
+                  - emptyDir: {}
+                    name: default-config-dir
+      clusterPermissions:
+        - rules:
+            - verbs:
+                - use
+              apiGroups:
+                - security.openshift.io
+              resources:
+                - securitycontextconstraints
+              resourceNames:
+                - privileged
+          serviceAccountName: rook-ceph-system
+      permissions:
+        - rules:
+            - apiGroups:
+                - ""
+              resources:
+                - configmaps
+              verbs:
+                - get
+                - list
+                - watch
+                - create
+                - update
+                - delete
+          serviceAccountName: rook-ceph-osd
+        - rules:
+            - apiGroups:
+                - ""
+              resources:
+                - pods
+                - services
+                - configmaps
+                - nodes
+                - nodes/proxy
+              verbs:
+                - get
+                - list
+                - watch
+            - apiGroups:
+                - batch
+              resources:
+                - jobs
+              verbs:
+                - get
+                - list
+                - watch
+                - create
+                - update
+                - delete
+            - apiGroups:
+                - ceph.rook.io
+              resources:
+                - "*"
+              verbs:
+                - "*"
+          serviceAccountName: rook-ceph-mgr
+        - rules:
+            - apiGroups:
+                - ""
+              resources:
+                - pods
+                - configmaps
+                - events
+                - persistentvolumes
+                - persistentvolumeclaims
+                - endpoints
+                - secrets
+                - pods/log
+                - services
+              verbs:
+                - get
+                - list
+                - watch
+                - patch
+                - create
+                - update
+                - delete
+            - apiGroups:
+                - apps
+              resources:
+                - daemonsets
+                - replicasets
+                - deployments
+              verbs:
+                - get
+                - list
+                - watch
+                - create
+                - update
+                - delete
+            - apiGroups:
+                - ""
+              resources:
+                - nodes
+                - nodes/proxy
+              verbs:
+                - get
+                - list
+                - watch
+            - apiGroups:
+                - storage.k8s.io
+              resources:
+                - storageclasses
```

Comment:
I get the following error when installing this operator with the AllNamespaces installMode:
```
E0314 22:28:52.263508       7 reflector.go:134] github.com/kubernetes-sigs/sig-storage-lib-external-provisioner/controller/controller.go:602: Failed to list *v1.PersistentVolumeClaim: persistentvolumeclaims is forbidden: User "system:serviceaccount:openshift-operators:rook-ceph-system" cannot list resource "persistentvolumeclaims" in API group "" at the cluster scope
```
@alecmerdler @ecordell

Reply:
I believe we are [tracking this here](https://jira.coreos.com/browse/OLM-939).